<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLobbyWindow</class>
 <widget class="QDialog" name="GameLobbyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Lobby</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #1a1d23, stop:1 #0f1114);
}

QLabel {
    color: #e8eaed;
    font-size: 11pt;
}

QLabel#titleLabel {
    color: #ffffff;
    font-size: 18pt;
    font-weight: bold;
    padding: 10px;
}

QLabel#gameNameLabel {
    color: #14a5a5;
    font-size: 14pt;
    font-weight: bold;
}

QLabel#roleLabel {
    color: #ffcc00;
    font-size: 12pt;
    font-weight: 600;
}

QLabel#gameIdLabel, QLabel#playerIdLabel {
    color: #888888;
    font-size: 10pt;
}

QFrame#infoFrame {
    background-color: #23262e;
    border: 2px solid #3a3f4a;
    border-radius: 10px;
    padding: 10px;
}

QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2d3139, stop:1 #23262e);
    color: #e8eaed;
    border: 2px solid #3a3f4a;
    border-radius: 8px;
    padding: 10px 20px;
    font-size: 12pt;
    font-weight: 600;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #3a3f4a, stop:1 #2d3139);
    border: 2px solid #4a5060;
    color: #ffffff;
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #1d2027, stop:1 #23262e);
}

QPushButton:disabled {
    background-color: #23262e;
    color: #5a5e66;
    border: 2px solid #2a2d36;
}

QPushButton#startGameButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #0d7377, stop:1 #0a5a5e);
    font-weight: bold;
}

QPushButton#startGameButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #14a5a5, stop:1 #0d7377);
}

QPushButton#startGameButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2d3139, stop:1 #23262e);
    color: #5a5e66;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Lobby :)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="infoFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="infoLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="gameNameLabel">
        <property name="text">
         <string>Game Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="roleLabel">
        <property name="text">
         <string>Your Role</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameIdLabel">
        <property name="text">
         <string>Game ID: 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="playerIdLabel">
        <property name="text">
         <string>Your Player ID: 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="startGameButton">
       <property name="text">
        <string>â–¶ Start Game</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
